core/src/autogluon/core/models/ensemble/bagged_ensemble_model.py

```diff
@@ -356,6 +357,32 @@ def predict_proba(self, X, normalize=None, **kwargs):
 
         return pred_proba
 
+    def predict_folds(self, X, **kwargs):
+        y_pred_probas = self.predict_proba_folds(X, **kwargs)
+        y_preds = []
+        for y_pred_proba in y_pred_probas:
+            y_pred = get_pred_from_proba(y_pred_proba=y_pred_proba, problem_type=self.problem_type)
+            y_preds.append(y_pred)
+        return y_preds
+
+    def predict_proba_folds(self, X, normalize=None, **kwargs):
```

Comment:
Is this method available also in classification or only in regression? If it is available in regression, would would it return?